ksqldb-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlan.java

```diff
@@ -25,7 +25,7 @@
   private final QueryId queryId;
   private final ExecutionStep<?> physicalPlan;
 
-  PhysicalPlan(
+  public PhysicalPlan(
```

Comment:
Oooo, this is probably the biggest decision being made here.  

Are there any options besides making this public?

Reply:
What is the concern?

Btw: I might actually want to rename this, as the `PhysicalPlan` is actually an "execution plan" (ie, one level below the physical plan), and I might want to add an actual (new) "physical plan" layer. It's a little be TBD to figure out the details.

Reply:
I was just trying to figure out if we want this as part of the public API or we consider it internal.